ui/page/components/account_selector.go

```diff
@@ -200,34 +204,35 @@ func (as *AccountSelector) Layout(gtx layout.Context) layout.Dimensions {
 }
 
 func (as *AccountSelector) ListenForTxNotifications(ctx context.Context) {
+	as.WL.MultiWallet.AddTxAndBlockNotificationListener(as.TxAndBlockNotificationListener, true, AccoutSelectorID)
```

Comment:
I'd say do `as.TxAndBlockNotificationListener = listeners.NewTxAndBlockNotificationListener()` here instead of in `NewAccountSelector`. The notif channel may have been closed if this method was previously called with a ctx that was canceled. Also, should ensure that either `as.TxAndBlockNotificationListener == nil` or the channel is closed before recreating/assigning `as.TxAndBlockNotificationListener` to avoid starting a fresh listener when the previous one hasn't been closed.